input/pagecontent/specifications_techniques-cpts-recherche_creneaux.md

```diff
@@ -0,0 +1,564 @@
+Afin de tester un fichier, il suffit de sélectionner le format `JSON`, d'ajouter le fichier via le bouton `Add…`, de sélectionner le modèle `BundleAgregateurCPTS` puis de cliquer sur `valider` :
+
+<table align="center">
+    <tr>
+        <td align ="center">
+            <div class="figure">
+                <img src="validateur_creneaux-CPTS_1.png" alt="Accès au validateur agrégateur de créneaux" title="Accès au validateur agrégateur de créneaux">
+            </div>
+        </td>    
+    </tr>
+    <tr>
+        <td align ="center">
+            <b>Figure 1 - Accès au validateur agrégateur de créneaux</b>
+        </td>
+    </tr>
+</table>
+
+Vous obtiendrez alors un rapport de test mettant en valeur les erreurs bloquantes et les différents warnings :
+
+<table align="center">
+    <tr>
+        <td align ="center">
+            <div class="figure">
+                <img src="validateur_creneaux-CPTS_2.png" alt="Rapport validateur agrégateur de créneaux" title="Rapport validateur agrégateur de créneaux">
+            </div>
+        </td>    
+    </tr>
+    <tr>
+        <td align ="center">
+            <b>Figure 2 - Rapport validateur agrégateur de créneaux</b>
+        </td>
+    </tr>
+</table>
+
+### Exemple de requête
+
+La requête ci-dessous correspond à une recherche de créneaux disponibles entre le 12 juin 2024 à 16h20 et le 15 juin 2024 à 16h20 pour les PS correspondant aux RPPS préfixés : 810002909371, 810001288385.
+
+**Requête :**
+
+`get[BASE]/Slot?_include=Slot:schedule&_include:iterate=Schedule:actor&_include=Slot:service-type-reference&_include:iterate=HealthcareService:organization&start=ge2024-06-12T16:20:00.000+02:00&start=le2024-06-15T16:20:00.000+02:00&schedule.actor:Practitioner.identifier=urn:oid:1.2.250.1.71.4.2.1|810002909371,urn:oid:1.2.250.1.71.4.2.1|810001288385&status=free`
+
+**Réponse simplifiée :**
+
+<details>
+  <summary>Déplier pour accéder au détail de l'exemple de réponse simplifiée</summary>
+  <pre>
+HTTP 200 OK
+  resourceType: Bundle
+  type: searchset
+  total: 1
+  Practitioner1
+  PractitionerRole1
+  Schedule1
+  Slot1
+  HealthcareService1
+  Organization1
+  </pre>
+</details>
+<br>
+
+**Corps de la réponse complète :**
+
+La réponse ci-dessous correspond à :
+1 créneau disponible avec Docteur Pierre FORET (RPPS : 810002909371)
+  - au 25 chemin de mounestie (31840) associé à la CPTS Axe Majeur (FINESS : 1950047225) 
+  - le 12 juin 2024 de 16h40 à 17h ayant les caractéristiques suivantes :
+    - Avec prise de RDV, 
+    - Visible du grand public et de la structure CPTS, 
+    - Au cabinet
+    - Pour le motif `Consultation`
+    - Téléphone de la CPTS : 0102030405
+
+<details>
+  <summary>Déplier pour accéder au détail de l'exemple de réponse complète au format json</summary>
+  <pre>
+{
```

Comment:
Il vaut mieux mettre les exemples en FSH directement comme a fait @breverseau 
Difficile de s'assurer que l'exemple est valide avec un copier coller, d'autant plus si on modifie les profils dans le temps

Reply:
Oui c'est prévu ainsi, mais dans l'attente que ce soit bien homogénéisé entre les cas d'usages (PS, CPTS).